module/zfs/vdev.c

```diff
@@ -2909,6 +2910,18 @@ vdev_get_stats_ex(vdev_t *vd, vdev_stat_t *vs, vdev_stat_ex_t *vsx)
 		if (vd->vdev_ops->vdev_op_leaf)
 			vs->vs_rsize += VDEV_LABEL_START_SIZE +
 			    VDEV_LABEL_END_SIZE;
+		/*
+		 * Report expandable space on top-level, non-auxillary devices
+		 * only.
+		 * The expandable space is reported in terms of metaslab sized
+		 * units
+		 * since that determines how much space the pool can expand.
+		 */
```

Comment:
[cstyle] Let's wrap this comment so it's a little more readable.

```+		/*
￼ +		 * Report expandable space on top-level, non-auxillary devices
￼ +		 * only. The expandable space is reported in terms of metaslab
￼ +		 * sized units since that determines how much space the pool
 +		 * can expand.
￼ +		 */
```